.github/workflows/push.yml

```diff
@@ -35,6 +38,7 @@ jobs:
         PYTHON_BLACK_CONFIG_FILE: pyproject.toml
         PYTHON_FLAKE8_CONFIG_FILE: .flake8
         PYTHON_ISORT_CONFIG_FILE: pyproject.toml
+
   build:
     continue-on-error: ${{ matrix.docker_from == 'alpine:edge' }}
```

Comment:
Comparing to `alpine:edge` is not needed anymore.

We should decide if builds against Netbox Prerelease or feature branch should mark the build as failed or not.

Reply:
I've changed it so that only `${{ matrix.build_cmd != './build-latest.sh' }}` will error.